lib/quests/json/make_me_drinks

```diff
@@ -0,0 +1,91 @@
+{
+  "id": "Q12R1",
```

Comment:
This should probably just be "Q12". This is the id for the quest as a whole. The other ids below are the for the quests requirements. They should probably be numbered Q12R1..Q12R3